data/_ui/tooltips.txt

```diff
@@ -1275,6 +1275,12 @@ tip "Extended jump effects"
 tip "Ship outlines in shops"
 	`Controls the display of your fleet ship icons when in the shipyard or outfitter. When fancy, applies a sobel filter to all ships. (This can result in high GPU load for large fleets.) When fast, just uses the ship's sprite.`
 
+tip "Flagship outline in HUD"
+	`Controls the display of your flagship icon in the HUD. When fancy, applies a sobel filter to the sprite. (This can result in high GPU load for some ships on very old or low performance computers.) When fast, just uses the ship's sprite.`
+
+tip "Target outline in HUD"
+	`Controls the display of your target ship icon in the HUD. When fancy, applies a sobel filter to the sprite. (This can result in high GPU load for some ships on very old or low performance computers.) When fast, just uses the ship's sprite.`
+
```

Comment:
I feel like this can be a single preference?

Reply:
You could even merge it with the other preference as well.

Reply:
I think at the very least these two new ones should be combined. I'd be fine with having these be separate from the outfitter/shipyard preference.